src/widgets/JSONForm/widgets/Select.js

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable react/forbid-prop-types */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Picker } from 'react-native';
+// eslint-disable-next-line import/no-unresolved
+import { Picker } from '@react-native-picker/picker';
```

Comment:
Not getting this linter warning for this one. Checkbox I was getting cause they seem to have screwed up their package json https://github.com/react-native-checkbox/react-native-checkbox/issues/111

Reply:
hm. it was complaining.. it's fine now though. thanks